internal/ingestion/task/debug_result_dsl_test.go

```
@@ -134,17 +177,29 @@ func TestBuildDebugResultDSL(t *testing.T) {
 	if bOutputs["output_format"].(map[string]any)["value"] != "text" {
 		t.Errorf("components.b output_format=%#v want {value:\"text\"}", bOutputs["output_format"])
 	}
+	if tt, _ := bOutputs["text"].(map[string]any)["type"]; tt != "<class 'str'>" {
```

golangci-lint (2.12.2)

[error] S1005: unnecessary assignment to the blank identifier

(staticcheck)